app/pathogen/arbovirus/dashboard/(visualizations)/country-seroprevalence-comparison-scatter-plot.tsx

```diff
@@ -0,0 +1,204 @@
+      }),
+    [stateFilteredDataWithEstimateNumbers]
+  );
+
+  const pathogensAvailable = useMemo(
+    () => uniq(rechartsData.map((dataPoint) => dataPoint.primaryKey)),
+    [rechartsData]
+  );
+  const countriesAvailable = useMemo(
+    () =>
+      uniq(
+        rechartsData.flatMap((dataPoint) =>
+          typedObjectKeys(dataPoint).filter((value) => value !== "primaryKey")
+        )
+      ),
+    [rechartsData]
+  );
+
+  if (pathogensAvailable.length > 1) {
+    return (
+      <div>
+        <p>
+          This graph cannot be shown when the data contains more than one
+          arbovirus. If you&apos;re interested in generating this graph, please use
+          the filter to the left to filter out data.
+        </p>
+      </div>
+    );
+  }
+
+  if (countriesAvailable.length > 2) {
+    return (
+      <div>
+        <p>
+          This graph cannot be shown when the data contains more than two
+          countries. If you&apos;re interested in generating this graph, please use
+          the filter to the left to filter out data.
+        </p>
+      </div>
+    );
+  }
+
+  const dataGroupedByCountry =
+    rechartsData[0] ?? countriesAvailable.map((country) => ({ [country]: [] }));
+
+  return (
+    <div className="w-full h-full flex">
+      {countriesAvailable.map((country, index) => (
+        <ResponsiveContainer
+          width={countriesAvailable.length === 1 ? "100%" : "50%"}
+          key={`country-seroprevalence-comparison-scatter-plot-${country}`}
+          height={"100%"}
+          className={index > 0 ? "pl-4" : ""}
+        >
+          <ScatterChart
+            width={730}
+            height={250}
+            margin={{ bottom: 40, left: 8, top: 50, right: 10 }}
+          >
+            <text
+              x='50%'
+              y={20}
+              fill="black"
+              textAnchor="middle"
+              dominantBaseline="central"
+            >
+              <tspan fontSize="20">{country}</tspan>
+            </text>
+            <CartesianGrid />
+            <XAxis
+              dataKey="seroprevalence"
+              type="number"
+              domain={[0, 100]}
+              unit="%"
+            >
+              <Label
+                value="Seroprevalence"
+                offset={-8}
+                position="insideBottom"
+              />
+            </XAxis>
+            <YAxis
+              dataKey="estimateNumber"
+              type="number"
+              domain={[0, stateFilteredDataWithEstimateNumbers.length + 1]}
+              allowDataOverflow={true}
+              label={{
+                value: "Study estimates",
+                angle: -90,
+                position: "insideLeft",
+              }}
+            />
+            <Tooltip content={CountrySeroprevalenceComparisonScatterPlotTooltip}/>
+            <Scatter data={dataGroupedByCountry[country]} fill="#000000">
+              <ErrorBar dataKey="seroprevalenceError" width={0} direction="x" />
```

Comment:
I'm aware that some of our calculated error stuff doesn't quite work and results in graphs that look like the one below. This is a data related problem and I'll make a post in basecamp hopefully tomorrow or something pointing out the bad data so that we can get that fixed but it's nothing to do with the graph so I'm not going to do anything about it in this PR.

![image](https://github.com/serotracker/dashboard/assets/86806388/11a640bc-862a-4bdf-b8f9-7978aa3d832c)

Reply:
Noted, I will look into this.

Reply:
Data should be fixed now

Reply:
Can you also add a note or "i" somewhere @SeanKennyNF? "95% confidence intervals were calculated using the Clopper-Pearson method if not reported in the source."

Reply:
> Data should be fixed now

Yup this looks much better, thank you so much Harriet!

> Can you also add a note or "i" somewhere `@SeanKennyNF`? "95% confidence intervals were calculated using the Clopper-Pearson method if not reported in the source."

Done :)
5f7631594f36a881a998009625aec097951d3272
![Peek 2024-03-30 12-35](https://github.com/serotracker/dashboard/assets/86806388/6a42b66d-fe55-4efa-9e37-e45bfbba8f68)

Reply:
This looks great, thank you! I like having the estimate and CI when the points are hovered over too. How feasible would it be to add a link to the study in the hover-over? Not absolutely necessary but might be helpful @SeanKennyNF